pkg/doc/verifiable/credential.go

```diff
@@ -374,6 +375,13 @@ func WithJWTPublicKeyFetcher(fetcher PublicKeyFetcher) CredentialOpt {
 	}
 }
 
+// WithUnsecuredJWT indicates the need to decode Verifiable Credential from unsecured JWT
+func WithUnsecuredJWT() CredentialOpt {
```

Comment:
I'm wondering if unsecured JWS should be detected in your `decodeRaw` method, and the option is rather about whether or not unsecured JWS are allowed (default to false).

Reply:
I also thought about this but decided to go the same way as for JWS - a user explicitly defines the way we need to decode VC. And if e.g. JWS decoding fails, we do not try to decode by just JSON unmarshalling (with built-in proof).

Reply:
@Baha-sk see https://github.com/hyperledger/aries-framework-go/pull/425#discussion_r332560468